pyccel/parser/semantic.py

```diff
@@ -9,6 +9,7 @@
 from collections import OrderedDict
 from itertools import chain
 
+from sympy import Tuple
```

Comment:
I'm confused. Isn't the goal of this PR to completely eradicate `sympy.Tuple` from Pyccel?

Reply:
It is used once in this file to create a temporary object in order to use the sympy atoms function. We cannot easily add or own version until we have removed most sympy strangeness. I think it should be possible after this PR.

Reply:
I see. @saidctb Can you have a quick look into this?

Reply:
it's only used here, https://github.com/pyccel/pyccel/blob/ee4c3259c42ef0533fedee1f96161df0f45fbb59/pyccel/parser/semantic.py#L2605
which for sure doesn't work properly anymore 
we need to use `_atomic` for that